cmd/controller/disk.go

```diff
@@ -42,6 +42,7 @@ type DiskInfo struct {
 	LogicalSectorSize uint32          // LogicalSectorSize is the Logical size of a disk in bytes
 	Compliance        string          // Compliance is implemented specifications version i.e. SPC-1, SPC-2, etc
 	DiskType          string          // DiskType represents the type of disk like Disk, Sparse etc.,
+	Size              string          // Size contains number of block/sector in a disk
```

Comment:
@sch00lb0y Shouldn't the variable name should be something else than size , such as NoOfBlocks or something else ?